lib/modules/commentTools.js

```diff
@@ -259,6 +259,12 @@ module.go = () => {
 		});
 	}
 
+	// slideEditBar on comments only
+	$body.on('input', 'form[id^=\'commentreply\'] textarea', function() {
```

Comment:
Nope, calling the `slideEditBar` function in the `attachEditorToUsertext` function doesn't work for comments, which is why i added this line.

The `updateCounter `function which was already in the code did the same since it needs to be called on comments too, exactly like mine.
That's also why i didn't edit the `makeEditBar `function, i fiddled around with it for a while, but then i saw the `updateCounter `function and i just decided to conform my code to that.

Editing the `makeEditBar `could also allow to move the big editor button, i tried to but then had some problem with the caching/cloning/_interesting things_/???. (See the comment to andytuba)